DatadogLogs/Sources/Log/LogEventEncoder.swift

```diff
@@ -44,21 +46,24 @@ public struct LogEvent: Encodable {
         // The Log error stack
         public var stack: String?
     }
+
     public struct DeviceInfo: Codable {
         // The architecture of the device
         public let architecture: String
+    }
 
-        enum CodingKeys: String, CodingKey {
-            case architecture = "architecture"
-        }
+    public struct OperatingSystem: Codable {
```

Comment:
public struct, please add docs

```suggestion
    /// ....
    public struct OperatingSystem: Codable {
```